Gruntfile.js

```diff
@@ -210,11 +188,12 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-bumpup');
   grunt.loadNpmTasks('grunt-tagrelease');
   grunt.loadNpmTasks('grunt-saucelabs');
+  grunt.loadNpmTasks('grunt-webpack');
 
-  grunt.registerTask('build', ['replace', 'jshint', 'concat', 'uglify']);
+  grunt.registerTask('build', ['replace', 'jshint', 'uglify:prewebpack', 'webpack', 'uglify:dist']);
```

Comment:
I believe you can also have Webpack do the uglification for you: http://webpack.github.io/docs/list-of-plugins.html#uglifyjsplugin